### PR TITLE
fuse  channel-wise mul or add into batchnorm operator

### PR DESCRIPTION
**Pull request by zakk0610**
Optimization pass to fuse channel-wise mul or add operator with batch normalization operator

Before:
    Y = BatchNorm(X, scale, bias, mean, var)
    Z = MUL(Y, A, broadcast=1, axis=1)
After
   Y = BatchNorm(X, scale, bias, mean, var)

Before:
    Y = BatchNorm(X, scale, bias, mean, var)
    Z = ADD(Y, A, broadcast=1, axis=1)
After
    Y = BatchNorm(X, scale, bias, mean, var)

### CODE REVIEW

**Comment by zakk0610** [comment]
@fumihwh Sorry, this PR is duplicated.
